### PR TITLE
flag archive update

### PR DESCRIPTION
**Pull request by darkmoonfirelyte**

### CODE REVIEW

File: FF1Blazorizer/Pages/ResFlags.razor [hunk 1]
```diff
@@ -80,7 +110,7 @@
 		<h4>Winter 2021 Duck Derby</h4>
 		<ul>
 			<li><a href="https://4-0-1.finalfantasyrandomizer.com/?s=00000000&f=Av0px2aVm9JRW18fLJI5CyrQi925lkAzxzaNAZisZogB2XUkQSHCkOiazw3xmVazz95CPu.TAxfFV9zc3JwGOCV1r9Xy28D.AucC">Pravoka Square</a></li>
-			<li><a href="https://4-0-1.finalfantasyrandomizer.com/?s=00000000&f=ADLdunoG4Xr6a291PjlbVrezVO893q7hK0UOLcMbdAB9WjK90feBfqH.Ukpa1hqUa236yYTllBrInCWce8.EOCV1r9Xy28D.AucC">Garland's Castle</a></li>
+			<li><a href="https://4-0-1.finalfantasyrandomizer.com/?s=00000000&f=ADLdunoG4Xr6a291PjlbVrezVO893q7hK0UOLcMbdAB9WjK90feBfqH.Ukpa1hqUa236yYTllBrInCWce8.EOCV1r9Xy28D.AucC">Garland's Castle</li>
```
Comment (tetron) [inline FF1Blazorizer/Pages/ResFlags.razor:113]: this seems wrong?

File: FF1Blazorizer/Pages/ResFlags.razor [hunk 2]
```diff
@@ -2,9 +2,39 @@
 <div class="content px-4 tinted">
 	<div class="nes-container is-dark col-xs-12">
 		<h1>FFR Community Resources</h1>
+    
+		<h3>2022</h3>
+		<h4>Winter 2022 Duck Derby</h4>
+		<ul>
+			<li><a href="https://4-2-3.finalfantasyrandomizer.com/?s=00000000&f=AEdU-w3IQRxI9hE498MawW05ICNdrS90yqScQYovpLYRAFrD.HToBuF-0wdZqanBDZ8Airr4TtIqGLSfo9E3woVTXWwU.gTkiO9sUSq275IcEfJ3yMorKRy81nEzn-laeOWsdkIwEx2bSd9tB">Coneria Circuit</a></li>
+			<li><a href="https://4-2-3.finalfantasyrandomizer.com/?s=00000000&f=otglMjanJ69y7bVDgVEEc2V3Uupv2tciWzM7Uy1HVZ987XxH9dU6jNqZxVDhQl5mZCQM6fKdOWmMYKwKgQbN8OHIRsaVa1zWOChfgYXmie89ozerMP-MdGIkcsutgXpk.9pYdkIwEx2bSd9tB">AMUT City</li>
```
Comment (tetron) [inline FF1Blazorizer/Pages/ResFlags.razor:10]: missing closing </a>

File: FF1Blazorizer/Pages/ResFlags.razor [hunk 3]
```diff
@@ -2,9 +2,39 @@
 <div class="content px-4 tinted">
 	<div class="nes-container is-dark col-xs-12">
 		<h1>FFR Community Resources</h1>
+    
+		<h3>2022</h3>
+		<h4>Winter 2022 Duck Derby</h4>
+		<ul>
+			<li><a href="https://4-2-3.finalfantasyrandomizer.com/?s=00000000&f=AEdU-w3IQRxI9hE498MawW05ICNdrS90yqScQYovpLYRAFrD.HToBuF-0wdZqanBDZ8Airr4TtIqGLSfo9E3woVTXWwU.gTkiO9sUSq275IcEfJ3yMorKRy81nEzn-laeOWsdkIwEx2bSd9tB">Coneria Circuit</a></li>
+			<li><a href="https://4-2-3.finalfantasyrandomizer.com/?s=00000000&f=otglMjanJ69y7bVDgVEEc2V3Uupv2tciWzM7Uy1HVZ987XxH9dU6jNqZxVDhQl5mZCQM6fKdOWmMYKwKgQbN8OHIRsaVa1zWOChfgYXmie89ozerMP-MdGIkcsutgXpk.9pYdkIwEx2bSd9tB">AMUT City</li>
+			<li><a href="https://4-2-3.finalfantasyrandomizer.com/?s=00000000&f=gIGP.bmH9HENIp-kvc-ufUOGY-tKo-P9rrGrPihS2Drdw9PRG48teROXN4o2Vof3V6QW9tzstx.V.0y2lGQoRer4-uY-v5d1MVBUnbGvA9Wz9Zukrrc1bzTAJQZwLbmsAwYrdkIwEx2bSd9tB">Melmond Ruins</li>
```
Comment (tetron) [inline FF1Blazorizer/Pages/ResFlags.razor:11]: missing closing </a>

**Review by tetron — APPROVED** [review]

**Comment by darkmoonfirelyte** [comment]
I made changes.

**Pull request merged**